am-lib/src/main/java/uk/gov/hmcts/reform/amlib/internal/utils/SecurityClassifications.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.amlib.internal.utils;
+
+import uk.gov.hmcts.reform.amlib.enums.SecurityClassification;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+import static java.util.stream.Collectors.toSet;
+
+public final class SecurityClassifications {
+
+    private SecurityClassifications() {
+        //NO-OP
+    }
+
+    /**
+     * Builds a set of security classifications including and below a given hierarchy.
+     *
+     * @param securityClassificationForRole integer value of security classification defined in
+     * {@link SecurityClassification}
+     * @return a set of security classifications
+     */
+    public static Set<SecurityClassification> fromValueOf(int securityClassificationForRole) {
```

Comment:
I'd think better name. Maybe `getSecurityClassificationsAtSameOrLowerHirarchy` or `getVisibleSecurityClassifications`

Same way we should seek for better name for argument. It is not important what input security classification is or comes from. Utility method should be generic.